demos/action_recognition_demo/python/action_recognition_demo.py

```diff
@@ -97,23 +98,28 @@ def main():
 
     encoder_xml = args.m_encoder
     encoder_bin = args.m_encoder.replace('.xml', '.bin')
-    encoder = IEModel(encoder_xml, encoder_bin, ie, encoder_target_device,
-                      num_requests=(3 if args.device == 'MYRIAD' else 1))
-
-    if args.m_decoder is not None:
+    model = []
```

Comment:
I think it should be renamed to `models`

Reply:
Changed